src/citrine/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '1.10.0'
+__version__ = '1.10.1'
```

Comment:
```suggestion
__version__ = '1.11.0'
```
You are changing a default behavior which means this is a potentially breaking change.  Even if you don't change the default, you are still arguably making an API change, which would still require a minor bump.